test/Microsoft.DotNet.Docker.Tests/DockerHelper.cs

```diff
@@ -0,0 +1,105 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.Docker.Tests
+{
+    public class DockerHelper
+    {
+        private string DockerOS => GetDockerOS();
+        public string ContainerWorkDir => IsLinuxContainerModeEnabled ? "/sandbox" : "c:\\sandbox";
+        public bool IsLinuxContainerModeEnabled => string.Equals(DockerOS, "linux", StringComparison.OrdinalIgnoreCase);
+        private ITestOutputHelper Output { get; set; }
+
+        public DockerHelper(ITestOutputHelper output)
+        {
+            Output = output;
+        }
+
+        public void Build(string dockerfilePath, string fromImage, string tag, string buildArgs)
+        {
+            string dockerfileContents = File.ReadAllText(dockerfilePath);
+            dockerfileContents = dockerfileContents.Replace("$base_image", fromImage);
+            dockerfilePath = dockerfilePath + ".temp";
```

Comment:
Instead of `dockerfilePath = dockerfilePath + ".temp";`

`string tempFilePath = dockerfilePath + ".temp";` or similar would make it more clear.

Reply:
Changed to tempDockerfilePath.  FYI - this code will be removed as soon as the CI machines are upgraded to a version of Docker which supports multi-stage builds.  This will allow the usage of ARGs in the FROM.  It will make this code so much simpler.

Reply:
Ok...thanks.